auth/jwt/middleware_test.go

```diff
@@ -74,7 +74,7 @@ func TestJWTParser(t *testing.T) {
 		return key, nil
 	}
 
-	parser := NewParser(keys, method, jwt.MapClaims{})(e)
+	parser := NewParser(keys, method, func() jwt.Claims { return jwt.MapClaims{} })(e)
```

Comment:
This anonymous func should be provided as a top-level function in the package, e.g.

```go
// MapClaimsFactory is a ClaimsFactory that returns an empty jwt.MapClaims.
func MapClaimsFactory() jwt.Claims {
    return jwt.MapClaims{}
}
```